README.md

```diff
@@ -1 +1,38 @@
-# my-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> This project is proposed by Microverse to prepare our professional portfolio from the first week.
+
```

Comment:
- [ ] [OPTIONAL] I think it's always good to add a screenshot in the readme. It will make your readme looks more professional and descriptive.